### PR TITLE
Move messages to external module

### LINKED ISSUE #1: Initialize message module
Move code connected with messages (format, parser, objects, etc.) from golem to this repository. Create module and integrate Golem with it.
There should be an interface that exposes two simple methods: parse and format.

### PR DESCRIPTION
**Pull request by jiivan**
fixes: https://github.com/golemfactory/golem-messages/issues/1

### CODE REVIEW

File: tests/golem/network/transport/test_message.py [hunk 1]
```diff
@@ -27,9 +24,11 @@ def slots(self):
         raise Exception()
 
 
-class TestMessages(unittest.TestCase, PEP8MixIn):
-    PEP8_FILES = ['golem/network/transport/message.py', ]
+fake_sign = lambda x: b'\000'*65
+fake_decrypt = lambda x: x
```
Comment (Elfoniok) [inline tests/golem/network/transport/test_message.py:28]: What problem this is solving?

File: golem/network/transport/tcpnetwork.py [hunk 2]
```diff
@@ -488,7 +489,7 @@ def connectionLost(self, reason=connectionDone):
 
     # Protected functions
     def _prepare_msg_to_send(self, msg):
-        ser_msg = msg.serialize()
+        ser_msg = msg.serialize(lambda x: b'\000'*message.Message.SIG_LEN)
```
Comment (Elfoniok) [inline golem/network/transport/tcpnetwork.py:492]: Why this is needed?

Comment (jiivan) [reply golem/network/transport/tcpnetwork.py:492]: I made `sign` argument mandatory, as it is always used in golem code. It was omitted only in tests.

File: tests/golem/network/transport/test_message.py (hunk 1 above)
Comment (jiivan) [reply tests/golem/network/transport/test_message.py:28]: Same as above. This test checks messaging not encryption, but arguments are mandatory.

**Review by Elfoniok — APPROVED** [review]

**Pull request merged**